force-app/main/default/classes/DonationHistoryController_TEST.cls

```diff
@@ -89,7 +89,7 @@ public class DonationHistoryController_TEST {
     */
     @IsTest
     private static void shouldThrowAccessException() {
-        User u = UTIL_UnitTestData_TEST.createUser('Minimum Access - Salesforce');
+        User standardUser = UTIL_UnitTestData_TEST.createUserWithoutInsert(UTIL_Profile.PROFILE_READ_ONLY);
```

Comment:
@JoseRossiSF and @dcaballeroSf 
FYI - see changes made to the `DonationHistoryController_TEST` class to fix test failures in the packaging org. No action on your part other than FYI.